game-server/data/handlers/instance/abyss/AbyssalSplinterInstance.java

```diff
@@ -147,11 +147,21 @@ public void handleUseItemFinish(Player player, Npc npc) {
 			case 701593: // Artifact of Protection (Hard Mode)
 				sendMsg(SM_SYSTEM_MESSAGE.STR_MSG_IDAbRe_Core_NmdDH_Wakeup());
 				spawn(216960, 329.70886f, 733.8744f, 197.60938f, (byte) 0);
+				if (player.getRace() == Race.ELYOS) {
+					spawn(700857, 326.1821f, 766.9640f, 202.1832f, (byte) 100, 79); // artifact of protection(elyos) for quest 30255
+				} else if (player.getRace() == Race.ASMODIANS) {
+					spawn(700858, 326.1821f, 766.9640f, 202.1832f, (byte) 100, 79); // artifact of protection(asmodians) for quest 30355
+				}
```

Comment:
```suggestion
				int artifactOfProtection = player.getRace() == Race.ELYOS ? 700857 : 700858; // for quest 30255 / 30355
				spawn(artifactOfProtection, 326.1821f, 766.9640f, 202.1832f, (byte) 100, 79);
```
Always prefer descriptive variable names to comments